pr-deploy/src/github.ts

```diff
@@ -14,11 +14,24 @@
  * limitations under the License.
  */
 
+import {GetResponseTypeFromEndpointMethod as GetType} from '@octokit/types';
 import {Octokit} from '@octokit/rest';
-import {GitHubAPI} from 'probot/lib/github';
 
-
-const ACTION: Octokit.ChecksUpdateParamsActions = {
+const octokit = new Octokit();
+type ChecksCreateParams = GetType<typeof octokit.ChecksCreateParams>;
+type ChecksListForRefResponseCheckRunsItem = GetType<
+  typeof octokit.ChecksListForRefResponseCheckRunsItem
+>;
+type ChecksListForRefResponseCheckRunsItemOutput = GetType<
+  typeof octokit.ChecksListForRefResponseCheckRunsItemOutput
+>;
+type ChecksListForRefParams = GetType<typeof octokit.ChecksListForRefParams>;
+type ChecksUpdateParams = GetType<typeof octokit.ChecksUpdateParams>;
+type ChecksUpdateParamsActions = GetType<
+  typeof octokit.ChecksUpdateParamsActions
+>;
```

Comment:
I'm confused about this - can't you refer to Octokit.SubType directly anymore? :(

Reply:
So this was all wrong and was setting `any` for all types. Found https://github.com/octokit/rest.js/releases/tag/v17.0.0#typescript which explains how to use the official types. 

Fixed in this PR - https://github.com/ampproject/amp-github-apps/pull/1014